extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -91,14 +95,18 @@ void FlowFileRepository::flush() {
 }
 
 void FlowFileRepository::printStats() {
+  auto opendb = db_->open();
+  if (!opendb) {
+    return;
```

Comment:
Is this okay, shouldn't we log or throw?

Reply:
logging error on failed `open`